scripts/buildAndroidSnapshots.js

```diff
@@ -0,0 +1,84 @@
+/* @flow */
+
+/* eslint-disable no-console */
+
+const urlExists = require('url-exists');
+const child_process = require('child_process');
+
+const exec = (command, options) =>
+  child_process.execSync(command, {
+    stdio: 'inherit',
+    ...options,
+  });
+
+const DEFAULT_URL =
+  'http://trinerdis.cz:8000/repository/snapshots/com/trinerdis/skypicker/';
+
+if (!process.env.ANDROID_DEPLOYMENT_PASSWORD) {
+  throw Error('You need to pass ANDROID_DEPLOYMENT_PASSWORD.');
+}
+
+// Read native dependencies versions from package.json (maps, vector icons...)
+const buildPackage = require('../.build/package.json');
+
+// Get versions and remove ^ if necessary
+const getDependencyVersion = packageName => {
+  return buildPackage.dependencies[packageName].replace('^', '');
+};
+
+// Get rnkiwimobile version
+const RNKiwiMobileVersion = buildPackage.version;
+
+const deployToTrinerdis = (packageName, manualVersion, buildJS = false) => {
+  const version = manualVersion
+    ? manualVersion
+    : getDependencyVersion(packageName);
+
+  return new Promise((resolve, reject) =>
+    urlExists(
```

Comment:
Just a question - why is this necessary?

Reply:
So I talk this with @Blackhex and we would collapse trinerdis if we deploy every time we merge so I am checking if that url exists and just deploying when we change versions.